bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -29,6 +29,9 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
         date_partition_parameter={{ task.date_partition_parameter | format_optional_string }},
         {%+ endif -%}
         depends_on_past={{ task.depends_on_past }},
+        {%+ if task.arguments | length > 0 -%}
+        arguments={{ task.arguments | format_as_tuple }},
```

Comment:
Why do we need to pass through format_as_tuple here? We use lists elsewhere.

Reply:
`bigquery_etl_query` expects a tuple parameter here: https://github.com/mozilla/telemetry-airflow/blob/bb956639405d5ebc8a38a69dac79c689a7d34f6c/dags/utils/gcp.py#L467

Reply:
> `bigquery_etl_query` expects a tuple parameter here: https://github.com/mozilla/telemetry-airflow/blob/bb956639405d5ebc8a38a69dac79c689a7d34f6c/dags/utils/gcp.py#L467

I'd argue that's poor documentation. It looks like we actually just call `list(arguments)` on it later. Any iterable should be allowed.

I'm fine either way. You can simplify by letting this be a list or you can keep format_as_tuple to match the stated contract for the function.

Reply:
It's always good to keep things simple. List it is.